js/front/dataset/index.js

```diff
@@ -17,10 +17,10 @@ import LeafletMap from 'components/leaflet-map.vue';
 import FollowButton from 'components/buttons/follow.vue';
 import FeaturedButton from 'components/buttons/featured.vue';
 import ShareButton from 'components/buttons/share.vue';
+import DiscussionThreads from 'components/discussions/threads.vue';
 
 // Legacy widgets
 import 'widgets/issues-btn';
-import 'widgets/discussions-btn';
 import 'widgets/integrate-btn';
```

Comment:
Oh it reminds me I have a legacy branch for issue btn and we need to port integrate too